### DISCUSSION TITLE
About FE catalog PROPERTY metadata_refresh_interval_sec

### DISCUSSION THREAD

**Discussion by calvin2020** [apache/doris#20489] [A - General / Q&A]
Hi:
My doris version is 1.2.2 and 1.2.3，I can use catalog property  "metadata_refresh_interval_sec" normally before,but recently when I create a  table in MySQL database,I can not get this table info via catalog, would you please help to check this issue?
thanks
My catalog info as below
CREATE CATALOG `mysql_jdbc_catalog` PROPERTIES (
"password" = "*XXX",
"driver_class" = "com.mysql.jdbc.Driver",
"checksum" = "80e4321ee33dd6670e3643d85bf5c7c0",
"driver_url" = "mysql-connector-java-5.1.47.jar",
"type" = "jdbc",
"user" = "xxxx",
"jdbc_url" = "jdbc:mysql://xx.xx.xx.xx:3306/gr_user_center?yearIsDateType=false&tinyInt1isBit=false&useCursorFetch=true",
"metadata_refresh_interval_sec" = "600"
);

**Comment by yagagagaga** [comment]
`metadata_refresh_interval_sec`  is a 2.0 property and cannot be used in 1.2.x.

You can use a shell script to call `REFRESH CATALOG mysql_jdbc_catalog` regularly.

**Reply by calvin2020** [reply]
> `metadata_refresh_interval_sec` is a 2.0 property and cannot be used in 1.2.x.
> 
> You can use a shell script to call `REFRESH CATALOG mysql_jdbc_catalog` regularly.

get it,thanks for your reply
